_posts/2014-02-15-geting-started-index.md

```diff
@@ -74,7 +74,7 @@ You'll also need to add the Sugar application to the `application/0` function in
 
 ```elixir
 def application do
-  [ applications: [ :sugar ],
+  [ applications: [],
```

Comment:
I'd prefer to either keep this as is or update this to be:

```elixir
def application do
  [extra_applications: [],
   mod: {YourProject, []}]
end
```

as seen with the latest version of Elixir. Reference: http://elixir-lang.org/getting-started/mix-otp/supervisor-and-application.html#the-application-callback